oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/register/ServiceInventory.java

```diff
@@ -58,7 +58,10 @@
     @Setter @Getter @Column(columnName = NAME, matchQuery = true) private String name = Const.EMPTY_STRING;
     @Setter @Getter @Column(columnName = IS_ADDRESS) private int isAddress;
     @Setter @Getter @Column(columnName = ADDRESS_ID) private int addressId;
-    @Setter(AccessLevel.PRIVATE) @Getter(AccessLevel.PRIVATE) @Column(columnName = NODE_TYPE) private int nodeType;
+    /**
+     * Node type of the service, default {@link NodeType#Normal}
+     */
+    @Setter(AccessLevel.PRIVATE) @Getter(AccessLevel.PRIVATE) @Column(columnName = NODE_TYPE) private int nodeType = 0;
```

Comment:
@kezhenxu94 As this type is int already, I think the value currently should be 0 already for all existing versions. I just change it for more clear and match the new type usage. That is all.